assets/charts/components/prometheus-operator/charts/grafana/Chart.yaml

```diff
@@ -1,7 +1,6 @@
 apiVersion: v1
-appVersion: 7.0.3
+appVersion: 7.2.0
```

Comment:
For some reason I'm seeing `Grafana v7.1.4 (82a235b54c)` in the Grafana dashboard.

Reply:
There was a manual override, which I have removed.